src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
```

Comment:
callback should bind to the context via `call` method and should has two params. The first param is context (this) and the second param is arguments (`number` in this case). And the function will be have access to `this` (to your object in this case) and property and change it. Also you don't need to assign the result of callback to the property